### ISSUE TITLE
Dynamics 365 templates not available in Power Pages portal by default

### ISSUE THREAD

**Issue by julians1966** [MicrosoftDocs/power-pages-docs#12]
Dynamics 365 Power Pages templates are not available in the https://make.powerpages.microsoft.com by default. The environment has Dynamics 365 Field Service installed.

The Dynamics 365 templates do appear but only after creating a Portal app with the Dynamics 365 template in the Power Apps Maker portal https://make.powerapps.com, but then you can't create a Power Pages site using the template as you are limited to one per environment 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d3733324-6797-ff19-82e7-1a1956e5412a
* Version Independent ID: d3733324-6797-ff19-82e7-1a1956e5412a
* Content: [Dynamics 365 templates](https://learn.microsoft.com/en-us/power-pages/templates/dynamics-365-templates)
* Content Source: [power-pages-docs/templates/dynamics-365-templates.md](https://github.com/MicrosoftDocs/power-pages-docs/blob/main/power-pages-docs/templates/dynamics-365-templates.md)
* Service: **power-pages**
* Sub-service: ****
* GitHub Login: @sampatn
* Microsoft Alias: **sampatn**
![Dynamics 365 templates missing](https://user-images.githubusercontent.com/29278645/205920696-8faf8b5c-0bbf-42dc-959a-e1f7ecfecd05.png)
![Dynamics 365 templates visible](https://user-images.githubusercontent.com/29278645/205920716-afbf5131-78c9-4f76-961b-d1244c2a8e2f.png)
![Dynamics 365 templates error](https://user-images.githubusercontent.com/29278645/205921160-c73ef3a9-d70c-42ea-9ecf-1de9fd3d836d.png)

**Comment by nickdoelman** [comment]
Thanks for the feedback, this issue be resolved. If it's still a problem, please submit a new issue or re-open this one.
